code/modules/html_interface/voting/voting.dm

```diff
@@ -239,6 +239,16 @@ var/global/datum/controller/vote/vote = new()
 	if(mode)
 		if(config.vote_no_dead && usr.stat == DEAD && !usr.client.holder)
 			return 0
+		if(mode == "map")
+			var/mob/M = usr
+			if(istype(M, /mob/new_player))
+				usr << to_chat(usr, "<span class='warning'>Only players that have joined the round may vote for the next map.</span>")
```

Comment:
wait is it meant to be usr << to_chat(usr,etc) ?

Reply:
Nope i'm a dumb

Reply:
oh geez I'm not awake enough to review code I guess